pkg/charts/cni/values.go

```diff
@@ -17,22 +17,28 @@ const (
 
 // Values contains the top-level elements in the cni Helm chart
 type Values struct {
-	InboundProxyPort    uint   `json:"inboundProxyPort"`
-	OutboundProxyPort   uint   `json:"outboundProxyPort"`
-	IgnoreInboundPorts  string `json:"ignoreInboundPorts"`
-	IgnoreOutboundPorts string `json:"ignoreOutboundPorts"`
-	CliVersion          string `json:"cliVersion"`
-	CNIPluginImage      string `json:"cniPluginImage"`
-	CNIPluginVersion    string `json:"cniPluginVersion"`
-	LogLevel            string `json:"logLevel"`
-	PortsToRedirect     string `json:"portsToRedirect"`
-	ProxyUID            int64  `json:"proxyUID"`
-	DestCNINetDir       string `json:"destCNINetDir"`
-	DestCNIBinDir       string `json:"destCNIBinDir"`
-	UseWaitFlag         bool   `json:"useWaitFlag"`
-	PriorityClassName   string `json:"priorityClassName"`
-	ProxyAdminPort      string `json:"proxyAdminPort"`
-	ProxyControlPort    string `json:"proxyControlPort"`
+	InboundProxyPort    uint          `json:"inboundProxyPort"`
+	OutboundProxyPort   uint          `json:"outboundProxyPort"`
+	IgnoreInboundPorts  string        `json:"ignoreInboundPorts"`
+	IgnoreOutboundPorts string        `json:"ignoreOutboundPorts"`
+	CliVersion          string        `json:"cliVersion"`
+	CNIPluginImage      string        `json:"cniPluginImage"`
+	CNIPluginVersion    string        `json:"cniPluginVersion"`
+	LogLevel            string        `json:"logLevel"`
+	PortsToRedirect     string        `json:"portsToRedirect"`
+	ProxyUID            int64         `json:"proxyUID"`
+	DestCNINetDir       string        `json:"destCNINetDir"`
+	DestCNIBinDir       string        `json:"destCNIBinDir"`
+	UseWaitFlag         bool          `json:"useWaitFlag"`
+	PriorityClassName   string        `json:"priorityClassName"`
+	ProxyAdminPort      string        `json:"proxyAdminPort"`
+	ProxyControlPort    string        `json:"proxyControlPort"`
+	Tolerations         []Tolerations `json:"tolerations"`
+}
+
+type Tolerations struct {
```

Comment:
nit: could keep `Tolerations` as the field name in the value struct and change the concrete type name to the singular form.

```suggestion
type Toleration struct {
```